includes/Admin/Admin_Page.php

```diff
@@ -272,6 +272,26 @@ public function admin_footer() {
 				'type' => 'text/template',
 			)
 		);
+
+		// Print check's page styles.
+		echo '<style>';
+		echo '
+		#plugin-check__results .notice {
+			margin-top: 20px;
+		}
+		#plugin-check__results .notice ~ h4:first-of-type {
+			margin-top: 20px;
+		}
+		#plugin-check__results > h4:first-of-type {
+			margin-top: 80.5px;
+		}
+		@media ( max-width: 782px ) {
+			#plugin-check__results > h4:first-of-type {
+				margin-top: 88.5px;
+			}
+		}
```

Comment:
A few suggestions to simplify this CSS (there's no need to use `first-of-type` or the `~` operator):
```suggestion
		#plugin-check__results .notice,
		#plugin-check__results .notice + h4 {
			margin-top: 20px;
		}
		#plugin-check__results h4:first-child {
			margin-top: 80.5px;
		}
		@media ( max-width: 782px ) {
			#plugin-check__results h4:first-child {
				margin-top: 88.5px;
			}
		}
```